include/exadg/incompressible_navier_stokes/spatial_discretization/operator_coupled.cpp

```diff
@@ -323,6 +328,45 @@ OperatorCoupled<dim, Number>::evaluate_nonlinear_residual(BlockVectorType &
   dst.block(1) *= -scaling_factor_continuity;
 }
 
+template<int dim, typename Number>
```

Comment:
This is very much similar to the rhs for the Stokes problem, which served as a basis here.